src/server/service/kv.rs

```diff
@@ -37,11 +37,11 @@ use kvproto::kvrpcpb::{self, *};
 use kvproto::raft_cmdpb::{CmdType, RaftCmdRequest, RaftRequestHeader, Request as RaftRequest};
 use kvproto::raft_serverpb::*;
 use kvproto::tikvpb::*;
-use prometheus::HistogramTimer;
 use tikv_util::collections::HashMap;
 use tikv_util::future::{paired_future_callback, AndThenWith};
 use tikv_util::metrics::HistogramReader;
 use tikv_util::mpsc::batch::{unbounded, BatchReceiver, Sender};
+use tikv_util::time::{duration_to_sec, Instant as TiInstant};
```

Comment:
Em, maybe we can use a better name instead of TiInstant 

/cc @BusyJay

Reply:
I renamed `tikv_util::time::Instant` as TiInstant here,because `std::time::Instant` is used somewhere else in this file. If `std::time::Instant` is ok to replaced by `tikv_util::time::Instant`, there would be no need to rename it. @siddontang

Reply:
Maybe a converter between `tikv_util::time::Instant` and `std::time::Instant` should be implemented?

Reply:
Newest `kv.rs` in master seems changed a lot, heavy use of `std::time::Instant` was moved to somewhere else. Seems I can just use `tikv_util::time::Instant` as `Instant` here now.